setup.py

```diff
@@ -1,14 +1,21 @@
-import os.path
 import setuptools
 
 with open('README.md', 'r') as fh:
     long_description = fh.read()
 
-with open(os.path.join(os.path.dirname(__file__), 'requirements.txt')) as fh:
-    install_requires = fh.readlines()
 
-with open(os.path.join(os.path.dirname(__file__), 'requirements-dev.txt')) as fh:
-    tests_require = fh.readlines()
+install_requires = ['requests==2.20.0', ]
+
+tests_require = [
```

Comment:
@jjaroszsfdc it looks like this commit reverts the changes from [#25](https://github.com/forcedotcom/SalesforcePy/pull/25/files) by @demianbrecht . Demian, anything you'd like us to consider before going through with this? Also is there a reason either of ye can see to maintain both the requirements files *and* the lists in setup.py? Finally Jakub, would you mind bumping the patch version number for this change